Added Update to CircleImageView Library

jaimehta19 opened the pull request:
<!--
  - Thanks for submitting code to Oppia! Please fill out the following as part of
  - your pull request so we can review your code more easily.
  -->

## Explanation
<!--
  - Explain what your PR does. If this PR fixes an existing bug, please include
  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
  - when this PR is merged.
  -->
Added update to the `CircleImageView` from version `3.0.1` to version `3.1.0` which was released on January 26th, 2020.

## Checklist
<!-- Please tick the relevant boxes by putting an "x" in them. -->
- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ ] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

Akash-Ramjyothi committed:
Added Update to Library

rt4914 commented on .idea/misc.xml:
```diff
@@ -1,9 +1,9 @@
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Revert changes to this file. These are unintentional changes.

rt4914 commented on .idea/runConfigurations.xml:
```diff
@@ -3,10 +3,11 @@
     </option>
   </component>
-</project>
+</project>
```
Revert changes to this file. These are unintentional changes.

rt4914 commented on app/build.gradle:
```diff
@@ -121,7 +121,7 @@ dependencies {
       'com.google.guava:guava:28.1-android',
       'com.github.oppia:CircularImageview:35d08ba88a',
-      'de.hdodenhof:circleimageview:3.0.1',
+      'de.hdodenhof:circleimageview:3.1.0',
```
You will need to apply this change in `third_party/versions.bzl` file too.

rt4914 reviewed (changes requested):
@jaimehta1912 PTAL Thanks.

rt4914 commented:
@jaimehta1912 Please make sure you have signed the CLA [here](https://github.com/oppia/oppia-android/wiki#onboarding-instructions).

jaimehta19 commented:
@rt4914  yes  sir  ,I have signed CLA already

rt4914 commented:
> @rt4914 yes sir ,I have signed CLA already

@jaimehta1912 Please reply to above comments and update the PR accordingly. Thanks.

BenHenning commented:
Thanks @jaimehta1912. FYI I'm closing this for the same reason mentioned in https://github.com/oppia/oppia-android/pull/3493#issuecomment-889642775. Similarly, if you think there's a reason why we should update this, then please let us know in a follow-up comment.

jaimehta19 commented:
@BenHenning The reason to Update this particular Library is that new features such as, support for image alpha, replaced fields holding bitmap dimensions with local variables, removed pointless override of getScaleType, restructured initialization and property updates and added support for animated drawables have been released with the Updated version of this Library.

jaimehta19 commented:
@rt4914
